README.md

```diff
@@ -1,4 +1,4 @@
-[![GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/unit_test_4gpu.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/unit_test_4gpu.yaml)
+[![GPU Integration Test](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_periodic.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_periodic.yaml)
```

Comment:
it makes sense to me to use the 'periodic' job for the status.

however i didn't realize we had a separate 'integration test' job vs our 'unit test' job.

would like to follow up on why they are separate or if it should just be the same set of tests.

Reply:
it looks like integration_test and unit_test_4gpu are identical except for when they run.

1) can we rename unit_test to integration test, that's what it really is
2) we can merge the yaml i think?  (just have 2 entries under the 'on:' field, one for PR and one for cron.